### ISSUE TITLE
Which classification loss function should I choose to enable me jointly minimize it with a prediction mean squared loss function in keras? I'm wondering if minimizing the sum of  a cross entropy with mse would make any sense at all.

### ISSUE THREAD

**Issue by amaall** [keras-team/keras#8355]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [ ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by ozabluda** [comment]
cross-entropy + mse together rarely makes sense, but if you want to, you can have two losses. Keras will minimize their (optionally weighted) sum.

**Comment by harshini-gadige** [comment]
Hi

This issue isn't related to a bug/enhancement/feature-request or other accepted types of issue.

To ask questions, please see the following resources :

https://keras.io/
https://keras.io/#support
https://gitter.im/Keras-io/Lobby

Thanks!

If you think I made a mistake, please open another issue.
